.changesets/exp_carton_ginger_magnet_beacon.md

```diff
@@ -0,0 +1,19 @@
+### Experimental: Introduce a pool of query planners ([PR #4897](https://github.com/apollographql/router/pull/4897))
+
+This changeset introduces an experimental pool of query planners to parallelize query planning.
+
+This feature is experimental, you can discuss it by following this link: https://github.com/apollographql/router/discussions/4917
+
+
+Configuration:
+
+```yaml
+supergraph:
+  query_planner:
+    experimental_available_parallelism: auto # number of available cpus
+```
+
+Note you can also set `experimental_available_parallelism` to a number representing how many planners you want to use in a pool.
+The default is `1`.
```

Comment:
```suggestion
This feature is experimental, you can discuss it by following this link: https://github.com/apollographql/router/discussions/4917

You can configure query planner pools by setting `supergraph.query_planner.experimental_available_parallelism`:

```yaml
supergraph:
  query_planner:
    experimental_available_parallelism: auto # number of available cpus
```

Its value is the maximum number of query planners that can run in parallel, and its default value is `1`. You can set it to the special value `auto` to automatically set it equal to the number of available CPUs. 
```